generic/tests/pktgen_perf.py

```diff
@@ -12,6 +12,7 @@
 from virttest import utils_misc
 from virttest import error_context
 
+from functools import partial as Partial
 
 _system_output = functools.partial(process.system_output, shell=True)
```

Comment:
Sorry, just notice this definition, so you actually can directly use this.

Reply:
@quanwenli Would you please update it? Thanks.

Reply:
update fix within https://github.com/autotest/tp-qemu/pull/2312